### ISSUE TITLE
Feature Request: Handle calendar invites within mail attachments (.ics)

### ISSUE THREAD

**Issue by mismith** [nylas/N1#169]
For instance, Mac Mail adds a toolbar to the top of the email view with buttons for "Accept", "Maybe", and "Decline". This simple shortcut saves a huge amount of time!

**Comment by alonisser** [comment]
+1, Option to handling ics would be great

**Comment by laughedelic** [comment]
+1

**Comment by bengotow** [comment]
Hey folks—we're working on this (there's actually some Event-related React code in there already ;-) ) and it should ship soon. I'll keep you posted.

**Comment by mzupan** [comment]
Here's a ics file if needed


```
BEGIN:VCALENDAR
METHOD:REQUEST
PRODID:Microsoft Exchange Server 2010
VERSION:2.0
BEGIN:VTIMEZONE
TZID:Pacific Standard Time
BEGIN:STANDARD
DTSTART:16010101T020000
TZOFFSETFROM:-0700
TZOFFSETTO:-0800
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=1SU;BYMONTH=11
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:16010101T020000
TZOFFSETFROM:-0800
TZOFFSETTO:-0700
RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=2SU;BYMONTH=3
END:DAYLIGHT
END:VTIMEZONE
BEGIN:VEVENT
ORGANIZER;CN=First Last:MAILTO:name@domain.com
ATTENDEE;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE;CN=First2 Last2
 co:MAILTO:name2@domain.com
ATTENDEE;ROLE=REQ-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP=TRUE;CN=First3 Last3
 ker:MAILTO:name3@domain.com
RRULE:FREQ=WEEKLY;INTERVAL=1;BYDAY=MO,TU,WE,TH,FR;WKST=SU
SUMMARY;LANGUAGE=en-US:Backend Standup
DTSTART;TZID=Pacific Standard Time:20151130T110000
DTEND;TZID=Pacific Standard Time:20151130T111000
UID:8B8BA553-3808-4E69-AEEA-6EBDCDCB2B93
CLASS:PUBLIC
PRIORITY:5
DTSTAMP:20151125T231706Z
TRANSP:OPAQUE
STATUS:CONFIRMED
SEQUENCE:0
LOCATION;LANGUAGE=en-US:Kitchen
X-MICROSOFT-CDO-APPT-SEQUENCE:0
X-MICROSOFT-CDO-OWNERAPPTID:2113654262
X-MICROSOFT-CDO-BUSYSTATUS:TENTATIVE
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MICROSOFT-CDO-ALLDAYEVENT:FALSE
X-MICROSOFT-CDO-IMPORTANCE:1
X-MICROSOFT-CDO-INSTTYPE:1
X-MICROSOFT-DISALLOW-COUNTER:FALSE
BEGIN:VALARM
DESCRIPTION:REMINDER
TRIGGER;RELATED=START:-PT5M
ACTION:DISPLAY
END:VALARM
END:VEVENT
END:VCALENDAR
```

**Comment by mzupan** [comment]
With that ics file I don't get any details of the event in the email. Only the option to download it

**Comment by bengotow** [comment]
Hey folks—"Quick RSVP" for emails with events shipped in 0.3.26. Enjoy!

**Comment by systemeintegratedsolutions** [comment]
+1. I dont seem to see the 'quick rsvp' you speak of. When I have an ICS attachment I also only have the option to download it. I know you are working on Google Calendar integration so eagerly waiting :D (again, you guys rock!). Would it be a quick enough change to allow the ICS to be pushed to my linked google account calendar in the meantime? (so basically without needing an interface on the Nylas app) ? Just asking, I know you have plenty to work on :)
